templates/confidential-computing/index.html

```diff
@@ -55,164 +68,154 @@ <h2>Benefits of Ubuntu for confidential computing</h2>
                 <div class="col-6 col-medium-3">
                   <p>
-                    When a confidential VM is launched, its integrity is verified and
-                    its initial code and data are measured by a hardware root of
-                    trust. This ensures they have not been tampered with. The
-                    measurement is cryptographically signed and can be attested to a
-                    remote verifier.
+                    We support a 6.8 kernel, derived from the 24.04 generic kernel, and offer essential user space components accessible through PPAs, such as Libvirt 9.6, and QEMU 8.0. We also offer a set of user-friendly scripts to simplify the creation of confidential environments with just a few commands.
+                  </p>
+                </div>
+              </div>
+            </li>
+            <li class="p-stepped-list__item">
+              <div class="row">
+                <div class="col-3 col-medium-3">
+                  <h3 class="p-heading-icon__title p-heading--5">Guest side</h3>
+                </div>
+                <div class="col-6 col-medium-3">
+                  <p>
+                    We support a comprehensive package, featuring a 6.8 kernel, Shim, Grub, and TDVF, which serves as an in-guest VM firmware.
                   </p>
                 </div>
               </div>
             </li>
           </ol>
+          <div class="row">
+            <div class="col-3 col-medium-3">
+            </div>
+            <div class="col-6 col-medium-3">
+              <hr class="p-rule--muted" />
+              <a href="https://github.com/canonical/tdx">Deploy Intel TDX with Ubuntu 24.04 today&nbsp;&rsaquo;</a>
+            </div>
+          </div>
         </div>
       </div>
     </section>
 
-    <hr class="p-rule--muted is-fixed-width" />
+    <section class="p-section">
+      <hr class="p-rule is-fixed-width" />
+      <div class="row--50-50">
+        <div class="col">
+          <h2>How confidential VMs work</h2>
+        </div>
+        <div class="col u-sv3">
+          <div class="row">
+            <p>
+              Confidential VMs introduce a new trust boundary which only includes the software running within, and the platform’s hardware. All other software outside is no longer part of your trusted computing base. To provide such strong security guarantees, confidential computing relies on two main primitives:
+            </p>
+          </div>
+          <div class="row u-sv3">
+            <div class="p-image__background--3-2 u-align--center u-vertically-center">
+              {{ image(url="https://assets.ubuntu.com/v1/480ebd6e-how-confidential-vms.png",
+                alt="",
+                width="1200",
+                height="800",
+                hi_def=True,
+                loading="auto|lazy"
+                ) | safe
+              }}
+            </div>
+          </div>
+          <div class="row">
+            <ol class="p-list--divided">
```

Comment:
`ol`s can only have `li`s as direct children to be valid html. The [vertical stepped list ](https://vanillaframework.io/docs/patterns/lists#vertical-stepped) with some added hrs is probably the best bet for this